server/controllers/errors/clientError.js

```diff
@@ -1,7 +1,5 @@
 const clientError = (req, res) => {
-  res
-    .status(404)
-    .send({ error: { code: 404, msg: 'Page Not Found' }, data: null });
+  res.status(404).redirect('/404');
```

Comment:
when I type any other route that it should redirect me to 404 page , it didn't do that so try to solve this problem .